homeassistant/components/climate/__init__.py

```diff
@@ -377,6 +377,14 @@ def _report_turn_on_off(feature: str, method: str) -> None:
             # Return if integration has migrated already
             return
 
+        supported_features = self.supported_features
+        if supported_features & (
+            ClimateEntityFeature.TURN_ON | ClimateEntityFeature.TURN_ON
+        ):
+            # The entity supports both turn_on and turn_off, the backwards compatibility
+            # checks are not needed
+            return
+
         supported_features = self.supported_features
```

Comment:
Duplicate fetch of the supported features property here

Reply:
#121619 fix + also the comment in the other PR